src/Viewer/components/StatusBar/StatusBar.js

```diff
@@ -75,28 +75,19 @@ export function StatusBar ({status, logFileState, loadingLogs, changeStateCallba
      * @return {string}
      */
     function generateLinkToLogEvent () {
-        const urlSearchParams = new VerbatimURLParams(window.location.search, "?");
-        urlSearchParams.set("prettify", String(logFileState.prettify));
-        const urlHashParams = new VerbatimURLParams(window.location.hash, "#");
-        if (logFileState.logEventIdx) {
-            urlHashParams.set("logEventIdx", logFileState.logEventIdx);
-        }
-        let url = `${window.location.origin}${window.location.pathname}`;
-        if (urlSearchParams.getLength()) {
-            url += `?${urlSearchParams}`;
-        }
-        if (urlHashParams.getLength()) {
-            url += `#${urlHashParams}`;
-        }
-        return url;
+        const hashParams = {
+            logEventIdx: logFileState.logEventIdx,
+        };
+
+        return getModifiedUrl({}, hashParams);
```

Comment:
This no longer adds the prettify param right?

Reply:
It does not change the `pretty` param if it is already in the URL.

We can think of the `searchParams` and `hashParams` as override lists, where specifying `<key>: null` in the lists means to delete all parameters matching `<key>` in the URL. 

Let me document these behaviours in the function JSDocs.

Reply:
Uh, but what I was noticing is that if you click prettify, it doesn't get added to the URL, but in the old version, if you clicked the copy link button, it would be in the copied URL; But now it isn't. Arguably that was confusing behaviour, so we could leave it as is. I just wanted to make sure the change was intentional.

Reply:
I see what you mean - we need to consider the case where `pretty` is / is not present in the URL initially.